packages/react-devtools-shared/src/__tests__/store-test.js

```diff
@@ -1932,9 +1932,8 @@ describe('Store', () => {
       );
 
       expect(store).toMatchInlineSnapshot(`
-        ✕ 1, ⚠ 0
         [root]
-          ▾ <Example> ✕
+          ▾ <Example>
```

Comment:
Yea. So this is directly related to this TODO:

https://github.com/facebook/react/blob/fe72cfcb2e70c211e1157b25b0ba5b299d28e95e/packages/react-reconciler/src/ReactChildFiber.js#L178-L180

Previously it was the "owner" of the parent which sometimes isn't related to where these elements are created at all. Now it's the actual child that should have the key specified on it but since that's a fake fiber it is not persistent in DevTools. We need to refactor ChildFiber to allow this warning to happen when we have access to the real fiber and it'll be better for context but I'm not planning on doing that yet because the rest of it is too much work already.